README.md

```diff
@@ -8,6 +8,8 @@
 <a href="https://coveralls.io/r/jstransformers/jstransformer?branch=master"><img src="https://img.shields.io/coveralls/jstransformers/jstransformer/master.svg" alt="Coverage Status"></a>
 <a href="https://www.npmjs.org/package/jstransformer"><img src="https://img.shields.io/npm/v/jstransformer.svg" alt="NPM version"></a></p>
 
+There are many good template engines, compilers, and transpilers written for node.js. But there is a problem: all of them have slightly different APIs, requiring slightly different usage. JSTransformer unifies them into one standardized API. Code written for one transformer will work with any other transformer. There are over 100 transformers, ranging from markdown parsers to template engines to coffeescript transpilers.
```

Comment:
> Is it necessary to write "compilers and transpilers"?

Perhaps not.

BTW, what is the proper term for markdown-to-html tools? Parsers? Compilers? Renderers?

> CoffeeScript isn't really that popular anymore. Maybe Babel?

Good point, IDK, just something that shows the diversity of JSTransformers.

Will definitely link to the keyword; thanks for the capitalization notes.